api/src/models/ImageFile.ts

```diff
@@ -76,15 +76,13 @@ class ImageFile {
             let ts_cmd = this.config().tesseractPath() + " " + deriv + " " + txt.slice(0, -4) + " " + this.ocrProperties();
             exec(ts_cmd, (error, stdout, stderr) => {
                 if (error) {
-                    throw `error: ${error.message}`;
-                    return;
+                    throw `error: ${error.message + " " + stderr}`;
                 } 
                 if (stderr) {
-                    throw `stderr: ${stderr}`;
-                    return;
+                    console.log(`stderr: ${stderr}`);
                 }
                 if (!fs.existsSync(txt)) {
-                    throw "Problem running textcleaner";
+                    console.log("Problem running textcleaner");
```

Comment:
This should still be a throw -- if the file we expected to create does not exist after all is said and done, something is very wrong and we should consider that an exception.